src/main/java/com/ezasm/gui/Window.java

```diff
@@ -328,4 +329,34 @@ public static void resetHighlight() {
         }
         Window.getInstance().editor.resetHighlighter();
     }
+
+    /**
+     * Connection function for passing the changed value
+     */
+    public static void passingValue(long number) {
+        if (instance == null || instance.table == null) {
+            return;
+        }
+        instance.table.addHighlightValue(number, instance.config);
```

Comment:
You're resetting the config every time. That's not needed, you only every use the copy of config you save to get a theme from it. We get the theme directly passed to the table class with `applyTheme`. Either save the whole theme there or just the colors you use from it. This will still properly update with changing the theme while the program is running.